week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -43,19 +43,69 @@ Outlook not so good.
 Very doubtful.
 */
 
+let array = [
+  // Very Positive:
+  "It is certain",
+  "It is decidedly so",
+  "Without a doubt",
+  "Yes - definitely",
+  "You may rely on it",
+  // Positive:
+  "As I see it, yes",
+  "Most likely",
+  "Outlook good",
+  "Yes",
+  "Signs point to yes",
+  // Negative:
+  "Reply hazy, try again",
+  "Ask again later",
+  "Better not tell you now",
+  "Cannot predict now",
+  "Concentrate and ask again.",
+  // very Negative:
+  "Don't count on it",
+  "My reply is no",
+  "My sources say no",
+  "Outlook not so good",
+  "Very doubtful."
+];
+
+function randomAnswer(array) {
+  return array[Math.floor(Math.random() * array.length)];
+}
+console.log(randomAnswer(array));
```

Comment:
It's good practice to remove your `console.log` statements after using them, otherwise we're leaving unnecessary statements in our scripts which might confuse other developers

Reply:
Removed.